pages/auth/redirect.vue

```diff
@@ -41,6 +49,16 @@ export default class RedirectPage extends Vue {
             method: method as string,
           })
           await this.initWallet(connection)
+          if (!this.currentAddress || !this.signer) return
+          const signature = await this.signMessageMemo('authorize', [
+            'profile',
+            'read:nftbook',
+            'write:nftbook',
+            'read:nftcollection',
+            'write:nftcollection',
+          ])
+          if (!signature) { return }
```

Comment:
Prefer throwing error for ui display

Reply:
[🥅 Throw connection error](https://github.com/likecoin/app-like-co/pull/474/commits/cc9826077540dfdcd12ebe0f57470e7419f8c922)